crates/holochain/src/conductor/handle.rs

```diff
@@ -984,7 +1036,7 @@ impl ConductorHandleT for ConductorHandleImpl {
             .await?;
 
         let roles = ops.role_assignments;
-        let app = InstalledAppCommon::new(installed_app_id, agent_key, roles);
+        let app = InstalledAppCommon::new(installed_app_id, agent_key, roles).unwrap();
```

Comment:
`.unwrap()` will panic, and we don't want that. if this type of error can't be returned here, you may have to use `.map_err`

```suggestion
        let app = InstalledAppCommon::new(installed_app_id, agent_key, roles)?;
```

Reply:
Absolutely, thanks for spotting. That was before I understood how to map errors.